lib/services/jira/jira_mapped_fields.rb

```diff
@@ -22,14 +21,15 @@ def custom_field_for_resource(resource, aha_field, jira_type_info)
     return nil unless resource.custom_fields # We only have custom fields for Requirements.
     
     field = resource.custom_fields.find {|field| field['key'] == aha_field}
+
     if field
-      aha_type_to_jira_type(field.value, field.type, jira_type_info, aha_field)
+      aha_type_to_jira_type(field.value, field.type, jira_type_info, aha_field, field)
     else
       nil
     end
   end
-  
-  def aha_type_to_jira_type(aha_value, aha_type, jira_type_info, aha_field)
+
+  def aha_type_to_jira_type(aha_value, aha_type, jira_type_info, aha_field, field)
```

Comment:
Do we call this method anywhere else? Should we default `field` to `nil` just in case?